webapp/src/components/AssetPage/ListingsTable/ListingsTable.spec.tsx

```diff
@@ -0,0 +1,249 @@
+    creator: '0xcreator',
+    beneficiary: null,
+    createdAt: 1671033414000,
+    updatedAt: 1671033414000,
+    reviewedAt: 1671033414000,
+    soldAt: 1671033414000,
+    data: {
+      parcel: undefined,
+      estate: undefined,
+      wearable: undefined,
+      ens: undefined,
+      emote: undefined
+    },
+    network: Network.MATIC,
+    chainId: ChainId.ETHEREUM_GOERLI,
+    firstListedAt: null
+  }
+
+  let ownersResponse: OwnersResponse = {
+    issuedId: 1,
+    ownerId: '0x92712b730b9a474f99a47bb8b1750190d5959a2b',
+    orderStatus: 'open',
+    orderExpiresAt: '1671033414000',
+    tokenId: '1'
+  }
+
+  let orderResponse: Order = {
+    id: '1',
+    marketplaceAddress: '0xmarketplace',
+    contractAddress: '0xaddress',
+    tokenId: '1',
+    owner: '0x92712b730b9a474f99a47bb8b1750190d5959a2b',
+    buyer: null,
+    price: '10',
+    status: ListingStatus.OPEN,
+    expiresAt: 1671033414000,
+    createdAt: 1671033414000,
+    updatedAt: 0,
+    network: Network.MATIC,
+    chainId: ChainId.ETHEREUM_GOERLI
+  }
+
+  afterEach(() => {
+    jest.resetAllMocks()
+  })
+
+  describe('Empty table', () => {
+    beforeEach(() => {
+      ;(nftAPI.nftAPI.getOwners as jest.Mock).mockResolvedValueOnce({
+        data: [],
+        total: 0
+      })
+      ;(orderAPI.orderAPI.fetchOrders as jest.Mock).mockResolvedValueOnce({
+        data: [],
+        total: 0
+      })
+    })
+
+    it('should render the empty table message', async () => {
+      const { getByTestId, findByTestId } = renderWithProviders(
+        <ListingsTable asset={asset} />
+      )
+
+      const loader = await findByTestId('loader')
+
+      await waitFor(() => {
+        expect(loader).not.toBeInTheDocument()
+      })
+
+      expect(getByTestId('empty-table')).toBeInTheDocument()
+    })
+  })
+
+  describe('Should render the table correctly', () => {
+    beforeEach(() => {
+      ;(nftAPI.nftAPI.getOwners as jest.Mock).mockResolvedValueOnce({
+        data: [ownersResponse],
+        total: 1
+      })
+      ;(orderAPI.orderAPI.fetchOrders as jest.Mock).mockResolvedValueOnce({
+        data: [orderResponse],
+        total: 1
+      })
+    })
+
+    it('should render the table', async () => {
+      const screen = renderWithProviders(<ListingsTable asset={asset} />)
+
+      const { findByTestId, getByTestId } = screen
+
+      const loader = await findByTestId('loader')
+
+      await waitFor(() => {
+        expect(loader).not.toBeInTheDocument()
+      })
+
+      expect(getByTestId('listings-table')).not.toBe(null)
+    })
+
+    it('should render the table data correctly', async () => {
+      const screen = renderWithProviders(<ListingsTable asset={asset} />)
+
+      const { findByTestId, getByTestId } = screen
+
+      const loader = await findByTestId('loader')
+
+      await waitFor(() => {
+        expect(loader).not.toBeInTheDocument()
+      })
+
+      expect(getByTestId(`profile-${orderResponse.owner}`)).not.toBe(null)
```

Comment:
```suggestion
      expect(getByText(orderResponse.owner).not.toBe(null)
```
Why don't we assert by the text here?
Same for all the strings down below